examples/nl-normal-address.html

```diff
@@ -190,10 +190,11 @@
 			"pricePickup": "5.85",
 			"pricePickupExpress": "1.38",
 
-			"deliveryTitle":"Bezorgen op",
+			"deliveryTitle":"Bezorgen",
 			"pickupTitle":"Afhalen op locatie",
 			"deliveryMorningTitle":"Ochtendlevering",
-			"deliveryStandardTitle":"Standaard levering",
+            "NLdeliveryStandardTitle":"Standaard levering",
```

Comment:
Probeer dit in 1 rechte lijn onder elkaar te zetten

Reply:
done